WalletWasabi/Models/TransactionBuilding/Cluster.cs

```diff
@@ -0,0 +1,48 @@
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using WalletWasabi.Bases;
+
+namespace WalletWasabi.Models
```

Comment:
```suggestion
namespace WalletWasabi.Models.TransactionBuilding
```

Reply:
Why?

Reply:
I meant you need to add `using WalletWasabi.Models.TransactionBuilding` in the files where you used the Cluster class (CoinsRegistry, CoinViewModel, TransactionProcessorTests, SmartCoin)

Reply:
Why? The code compiles. I don't understand what you say.

Reply:
The namespace in the Cluster.cs file should be `WalletWasabi.Models.TransactionBuilding`
If you change that you should add `using WalletWasabi.Models.TransactionBuilding` in the files where you used the Cluster class.

Reply:
Ohhh I see what you mean. I have moved the file out of the TransactionBuilding folder. Thank you.